filament/backend/include/private/backend/DriverAPI.inc

```diff
@@ -182,7 +182,8 @@ DECL_DRIVER_API_R_N(backend::BufferObjectHandle, importBufferObject,
         intptr_t, id,
         backend::BufferObjectBinding, bindingType,
         backend::BufferUsage, usage,
-        uint32_t, byteCount)
+        uint32_t, byteCount,
+        bool, takeOwnership)
```

Comment:
OpenGL backend could become the owner of imported buffers and textures unconditionally for our use case. It would result in a more lightweight diff compared to upstream. Also `takeOwnership = false` is not handled in Metal in this diff, and I think we shouldn't spend the effort on it. Ownership rules are already backend-dependent (due to the nature of OpenGL and Metal), we can tweak the rules of one of them as we see fit.

Reply:
[8caf282](https://github.com/shapr3d/filament/pull/93/commits/8caf2825ff5752648410c6fb50de9081fda03b1d)